src/terminal/terminal/scrollbar.h

```diff
@@ -260,6 +260,10 @@ void guac_terminal_scrollbar_flush(guac_terminal_scrollbar* scrollbar);
  * completely recreate and redraw the scrollbar rendering over the given
  * socket.
  *
+ * @deprecated guac_terminal_scrollbar_dup_batch() allows synchronizing
+ * multiple joining users at once using a broadcast socket, and should be
+ * used instead of this function
```

Comment:
Is there any need to maintain and deprecate this variant? Checking `Makefile.am`, this header looks to be part of the private API.

Reply:
Nope. I got rid of it.